Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -291,6 +291,18 @@ internal static void UpdateLevelKind(MatchExpression matchExpression)
         {
             AnalyzeContext context = base.CreateContext(options, logger, runtimeErrors, policy, filePath);
 
+            // Type or member is obsolete.
+#pragma warning disable CS0618
+            if (options.FileSizeInKilobytes != options.MaxFileSizeInKilobytes && options.FileSizeInKilobytes != default)
```

Comment:
>FileSizeInKilobytes != default [](http://example.com/codeflow?start=89&length=30)

what would happen if:
FileSizeInKilobyte = 1
MaxFileSizeInKilobyte = 2

Is that correct?
should we generate an exception/validation error?

Reply:
Also took advantage of Michael's recent update to Program.Main to validate the exact exception by retrieving the instantiated command.